### PR TITLE
Fix hardcoding of alpha variable in mix_up function in mixup.md and mixup.ipynb

### PR DESCRIPTION
**Pull request by uzair-ali10**
Followup PR of #438

### CODE REVIEW

**Comment by uzair-ali10** [comment]
@fchollet 
Kindly review this :)

**Review by fchollet — COMMENTED** [review]
Thanks for the PR. Please include all 3 files in one PR.

**Comment by uzair-ali10** [comment]
@fchollet 
Looks good now ??

File: examples/vision/ipynb/mixup.ipynb [hunk 1]
```diff
@@ -192,7 +192,7 @@
    "outputs": [],
    "source": [
     "\n",
-    "def sample_beta_distribution(size, concentration_0=0.2, concentration_1=0.2):\n",
+    "def sample_beta_distribution(size, concentration_0=alpha, concentration_1=alpha):\n",
```
Comment (fchollet) [inline examples/vision/ipynb/mixup.ipynb:195]: There's another line in this file that needs updating (same as the other files)

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**